pkg/autometrics/main.go

```diff
@@ -38,19 +38,25 @@ type Context struct {
 	TrackCallerName bool
 	// AlertConf is an optional configuration to add alerting capabilities to the metrics.
 	AlertConf *AlertConfiguration
-	// startTime is the start time of a single function execution.
-	// Only autometrics.Instrument should read this value.
-	// Only autometrics.PreInstrument should write this value.
+	// StartTime is the start time of a single function execution.
+	// Only amImpl.Instrument should read this value.
```

Comment:
Why can't this be called autometrics?

Reply:
I will add a mention with links to the other packages